Makefile

```diff
@@ -216,34 +216,34 @@ clean:
 	mkdir -p $(TOPDIR)/bin
 
 obj/image-crom.bin:
-	${LD} -o obj/image-crom.elf ${OBJECTS-CROM} ${RESOURCES} ${LDFLAGS-ROM}
-	${OBJCOPY} --output-target=binary --strip-all obj/image-crom.elf $@
+	@${LD} -o obj/image-crom.elf ${OBJECTS-CROM} ${RESOURCES} ${LDFLAGS-ROM}
```

Comment:
Above this are `resources` and `clean` targets, which still generate output. What was the deciding factor?

Reply:
The perl script produces no output besides a warning. It doesn't make sense (imo) to remove the output for clean.